src/libs/splashscreen/splashscreen.native.test.tsx

```diff
@@ -0,0 +1,50 @@
+import mockdate from 'mockdate'
+
+import { SplashScreenProvider, useSplashScreenContext } from 'libs/splashscreen/splashscreen'
+import { renderHook, act } from 'tests/utils'
+
+const MIN_SPLASHSCREEN_DURATION = 2000
+
+const TODAY = new Date('2022-10-14')
+
+jest.useFakeTimers()
+
+describe('useSplashScreenContext()', () => {
+  beforeEach(() => {
+    mockdate.set(TODAY)
+  })
+
+  it('should hide splashscreen without waiting when it has been shown for long enough', () => {
+    const { result } = renderSplashScreenHook()
+    mockdate.set(TODAY.getTime() + MIN_SPLASHSCREEN_DURATION)
+
+    act(() => {
+      result.current.hideSplashScreen && result.current.hideSplashScreen()
```

Comment:
pourquoi il y a besoin de cette condition ?

si c'est juste pour typescript, est-il possible de l'écrire comme ça ?

```suggestion
      result.current.hideSplashScreen.?()
```